composer.json

```diff
@@ -123,9 +123,6 @@
     "remove-git-submodules": "find . -mindepth 2 -type d -name .git | xargs rm -rf"
   },
   "config": {
-    "platform": {
-      "php": "7.4"
```

Comment:
This should be changed to 8.0. It basically ensures that when you run PHP 8.1 on your local and install/update a dependency then it will not install dependency that is for >= 8.2.

Reply:
@mollux are you OK to reverting this and just updating it to 8.0?